crates/query-engine/translation/src/translation/mutation/experimental/insert.rs

```diff
@@ -213,8 +210,10 @@ pub fn translate(
     arguments: &BTreeMap<String, serde_json::Value>,
 ) -> Result<(sql::ast::Insert, sql::ast::ColumnAlias), Error> {
     let object = arguments
-        .get("_objects")
-        .ok_or(Error::ArgumentNotFound("_objects".to_string()))?;
+        .get(&mutation.objects_argument_name)
+        .ok_or(Error::ArgumentNotFound(
```

Comment:
```suggestion
        .ok_or_else(|| Error::ArgumentNotFound(
```

Reply:
What is the difference?

Reply:
Went to have a look. It seems like `.ok_or_else` is useful when the or_else part is an expensive computation. This isn't the case here - we need to create a closure that returns this particular data, I think this will end up doing more work and require more memory, won't it? So isn't it strictly worse than just `ok_or` in this case?

Reply:
It's just because of the `.clone()`; I think it's probably borderline.